enhancements/update/upgrades-blocking-on-ack.md

```diff
@@ -0,0 +1,130 @@
+---
+title: upgrade-acknowledgment-gate
+authors:
+  - "@jottofar"
+reviewers:
+  - "@LalatenduMohanty"
+  - "@bparees"
+  - "@sdodson"
+  - "@csrwng"
+  - "@relyt0925"
+  - "@jeremyeder"
+approvers:
+  - TBD 
+creation-date: 2021-07-08
+last-updated: 2021-07-08
+status: implementable
+---
+
+# Upgrade Acknowledgment Gate
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+CVO currently blocks minor level upgrades when overrides are set, a resource deletion is pending, or an operator has set its upgradeable condition to false.
+This enhancement adds a generic upgrade blocking mechanism for blocking minor level upgrades.
+This generic upgrade blocker can consist of one or more items each of which will block a minor level upgrade.
+Each generic upgrade blocker item will be associated with some well-defined task to be completed by an administrator.
+Once the task has been completed the administrator will acknowledge completion and its associated item will no longer block minor level upgrades.
+
+## Motivation
+
+OCP 4.9 uses kubernetes v1.22 which is removing a large number of [APIs][k8s-1.22-removed-apis].
+We suspect that many customer clusters are still using these APIs through helm charts, operators, generic workloads, administrative tools, etc.
```

Comment:
I'd be sure to call out that we're specifically concerned about latent external use of APIs removed in 1.22. The cluster will generate alerts for active use.

Mostly just want to make it clear that we're not just being lazy, we believe there is no reasonable way for a Cluster Operator to provide a strong enough signal here.

Reply:
made some additions